src/PostgREST/Error.hs

```diff
@@ -196,6 +197,11 @@ instance JSON.ToJSON ApiRequestError where
     "message" .= ("Could not choose the best candidate function between: " <> T.intercalate ", " [pdSchema p <> "." <> pdName p <> "(" <> T.intercalate ", " [ppName a <> " => " <> ppType a | a <- pdParams p] <> ")" | p <- procs]),
     "details" .= JSON.Null,
     "hint"    .= ("Try renaming the parameters or the function itself in the database so function overloading can be resolved" :: Text)]
+  toJSON (ColumnNotFound colName) = JSON.object [
+    "code"    .= ApiRequestErrorCode18,
+    "message" .= ("Could not find '" <> colName <> "' in the target table" :: Text),
```

Comment:
Making the message resemble postgresql (`column projects.xxname does not exist"`):

```suggestion
    "message" .= ("Column '" <> colName <> "' does not exist" :: Text),
```

Adding the table name would be great too.

Reply:
I've made the change in message style and added the table name, both (separate from our talk about fuzzy matching above). Thanks!